stdlib/public/core/CollectionAlgorithms.swift

```diff
@@ -285,7 +291,8 @@ extension MutableCollection where Self: BidirectionalCollection {
   /// After partitioning a collection, there is a pivot index `p` where
   /// no element before `p` satisfies the `belongsInSecondPartition`
   /// predicate and every element at or after `p` satisfies
-  /// `belongsInSecondPartition`.
+  /// `belongsInSecondPartition`. This operation is not stable, so the
+  /// relative ordering of elements within the partitions is not preserved.
```

Comment:
Use a contraction, per reference style guidelines:

```suggestion
  /// `belongsInSecondPartition`. This operation isn't stable, so the
  /// relative ordering of elements within the partitions isn't preserved.
```